src/App.js

```diff
@@ -8,12 +8,23 @@ import LocationForm from './components/LocationForm';
 
 class App extends Component {
 
-    constructor() {
-        super();
+    state = {
```

Comment:
There isn't. This approach is more preferred but there are times when you need to do it in the constructor.